src/validation.cpp

```diff
@@ -1851,6 +1851,18 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         flags |= SCRIPT_VERIFY_NULLDUMMY;
     }
 
+    // SEGWIT2X signalling.
+    if ( VersionBitsState(pindex->pprev, chainparams.GetConsensus(), Consensus::DEPLOYMENT_SEGWIT2X, versionbitscache) == THRESHOLD_ACTIVE &&
```

Comment:
@jacob-eliosoff The purpose of an additional 2 weeks LOCKED_IN period as specified by BIP9, is that it allows miners who disagree with the upgrade, to upgrade when it is locked in.

If i disagree with this upgrade, i don't want to signal even if it is as simple as setting a flag in my stratum servers as I don't want my bit to be counted towards the 80%. 

I might want to change my mind when it turns out 80% does, in which case I don't want to be orphaned. To achieve this, a LOCKED_IN period is required, and 2 weeks is reasonable.